src/mame/drivers/gigatron.cpp

```diff
@@ -88,13 +94,13 @@ void gigatron_state::gigatron(machine_config &config)
 ROM_START( gigatron )
 	ROM_REGION( 0x20000, "maincpu", 0 )
 	ROM_SYSTEM_BIOS(0, "v4", "Gigatron ROM V4")
-	ROMX_LOAD( "ROMv4.rom",  0x0000, 0x20000, CRC(78995109) SHA1(2395fc48e64099836111f5aeca39ddbf4650ea4e),ROM_BIOS(0))
+	ROMX_LOAD( "gigrom4.rom",  0x0000, 0x20000, CRC(78995109) SHA1(2395fc48e64099836111f5aeca39ddbf4650ea4e),ROM_BIOS(0))
 	ROM_SYSTEM_BIOS(1, "v3", "Gigatron ROM V3")
-	ROMX_LOAD( "ROMv3.rom",  0x0000, 0x20000, CRC(1536efbe) SHA1(959268069e761a01d620396eedb9abc1ee63c421),ROM_BIOS(1))
+	ROMX_LOAD( "gigrom3.rom",  0x0000, 0x20000, CRC(38b177c1) SHA1(959268069e761a01d620396eedb9abc1ee63c421),ROM_BIOS(1))
 	ROM_SYSTEM_BIOS(2, "v2", "Gigatron ROM V2")
-	ROMX_LOAD( "ROMv2.rom",  0x0000, 0x20000, CRC(b4a3d936) SHA1(c93f417d589144b912c79f85b9e942d66242c2c3),ROM_BIOS(2))
+	ROMX_LOAD( "gigrom2.rom",  0x0000, 0x20000, CRC(45f4902c) SHA1(c93f417d589144b912c79f85b9e942d66242c2c3),ROM_BIOS(2))
 	ROM_SYSTEM_BIOS(3, "v1", "Gigatron ROM V1")
-	ROMX_LOAD( "ROMv1.rom",  0x0000, 0x20000, CRC(8ea5a2af) SHA1(e5758d5cc467c3476bd8f992fd45dfcdf06d0430),ROM_BIOS(3))
+	ROMX_LOAD( "gigrom1.rom",  0x0000, 0x20000, CRC(c6c9c09f) SHA1(e5758d5cc467c3476bd8f992fd45dfcdf06d0430),ROM_BIOS(3))
```

Comment:
These aren't the CRC32 hashes for the last ROMs you sent. The ones being replaced here look correct.

Reply:
the crc32 things were a mistake i believe in merging from here to mine, my bad.